Wild dog revision (#131)
Imageomics/kabr-tools: README.md

> # kabr-tools  [![DOI]()]()
> <!-- need new DOI for this release -->

Reviewer: ```suggestion
# kabr-tools   [![DOI](https://zenodo.org/badge/805519058.svg)](https://zenodo.org/doi/10.5281/zenodo.11288083)
```
This will be auto-updated with the new DOI

Our pipeline processes multi-species camera trap and drone data using object detection, tracking, and behavioral classification to generate five key metrics: time budgets, behavioral transitions, social interactions, habitat associations, and group composition dynamics. The modular design allows for integration of novel machine learning models and adaptation across species and study systems. We demonstrate the utility of kabr-tools through case studies on zebra time budgets, mixed-species social interactions, and African wild dog individual identification. By automating behavioral monitoring, kabr-tools enables scalable ecological research and conservation applications across diverse ecosystems.
</details>

## Overview

> ![](https://imageomics.github.io/kabr-tools/images/kabr-tools-visual-abstract.png)

Reviewer: ```suggestion
![](https://github.com/Imageomics/kabr-tools/raw/main/docs/images/kabr-tools-visual-abstract.png)
```

- Scan sampling
- Drone-based video analysis

#### See [Case Studies](https://imageomics.github.io/kabr-tools/case_studies/overview/) for examples demonstrating the application of kabr-tools in various ecological contexts, including:
- Grevy's zebra time budgets
- Mixed-species social interactions
- African wild dog individual behavioral monitoring

#### Example time budget analysis output comparing behavior granularity collected from drone videos versus traditional field observations.

> ![graph displaying the time budget comparison between drone-based behavior classification and manual field focal observations, demonstrates how drones allow for observations of more behaviors](https://imageomics.github.io/kabr-tools/images/timebuget_drone_manual.png)

Reviewer: ```suggestion
![graph displaying the time budget comparison between drone-based behavior classification and manual field focal observations, demonstrates how drones allow for observations of more behaviors](https://github.com/Imageomics/kabr-tools/raw/main/docs/images/timebuget_drone_manual.png)
```

## 🔗 License

This project is licensed under the [MIT License](https://github.com/Imageomics/kabr-tools/blob/main/LICENSE).

> ![](https://imageomics.github.io/kabr-tools/images/zebras_boxes.webp)

Reviewer: ```suggestion
![](https://github.com/Imageomics/kabr-tools/raw/main/docs/images/zebras_boxes.webp)
```
